documents-services/src/main/java/org/exoplatform/documents/rest/DocumentFileRest.java

```diff
@@ -1298,5 +1302,58 @@ public Response importDocuments(@Parameter(description = "upload id", required =
     }
   }
 
+  @GET
+  @Produces(MediaType.APPLICATION_JSON)
+  @RolesAllowed("administrators")
+  @Path("/deleted")
+  @Operation(summary = "Fetch trash documents", method = "GET", description = "Fetch trash document")
+  @ApiResponses(value = { @ApiResponse(responseCode = "200", description = "Request fulfilled"),
+      @ApiResponse(responseCode = "400", description = "Invalid query input"),
+      @ApiResponse(responseCode = "401", description = "Unauthorized operation"),
+      @ApiResponse(responseCode = "500", description = "Internal server error"), })
+  public Response getDeletedDocuments(@QueryParam("sortField")
+                                      String sortField,
+                                      @Parameter(description = "Sort ascending or descending")
+                                      @QueryParam("sortDirection")
+                                      String sortDirection,
+                                      @Parameter(description = "Offset of results to return")
+                                      @QueryParam("offset")
+                                      int offset,
```

Comment:
can we have a default value for offset ? (zero for example)

Reply:
done!